controllers/tweet-controller.js

```diff
@@ -1,4 +1,5 @@
 const { Tweet, User } = require('../models')
+// const helpers = require('../_helpers')
```

Comment:
The response is align to test files!
And...at line 2, what's the reason to include the helpers module but comment it out?

Reply:
Sorry that was something left while error finding when the test failed (but was pass in fact lol). Deleted.